book/website/collaboration/shared-ownership/shared-ownership-defaults.md

```diff
@@ -0,0 +1,54 @@
+(cl-shared-ownership-defaults)=
+# Nudging for a Better Default
+
+Research has shown that humans making decisions have a strong tendency to go along with the status quo or default option (See the [Nudge Theory](https://www.imperial.ac.uk/nudgeomics/about/what-is-nudge-theory/) by {cite}`ThalerSunstein2009Nudge`).
+In the absence of an actively defined default, the prevailing cultural or institutional norm becomes the de-facto option of choice.
+And hence, the default options for ongoing and new projects should nudge strongly towards adopting a shared ownership model.
+
+The minimum default for an open source project should include the following documents:
+i. defining the open source license,
```

Comment:
The ordered list items are rendered on the same line. Intuitively, the items should be rendered on separate lines.

Reply:
Fixed! Thanks Pooja.